src/shipit_static_analysis/tests/test_clang.py

```diff
@@ -1,13 +1,20 @@
 # -*- coding: utf-8 -*-
-from shipit_static_analysis.clang import ClangIssue
+from shipit_static_analysis.clang.tidy import ClangTidyIssue
+from shipit_static_analysis.clang.format import ClangFormat, ClangFormatIssue
+
+BAD_CPP_OLD = '#include <demo>\nint \tmain(void){\n printf("plop");return 42;    '
+BAD_CPP_NEW = '''#include <demo>
+int main(void) {
+  printf("plop");
+  return 42;'''
 
```

Comment:
Not: Please use the same string formatting in both constants to facilitate visual comparison.

Reply:
Not "Not" but "Nit". Thank you, auto-correct.

Reply:
:+1: